src/utils/bytesParsers/formatNumber.ts

```diff
@@ -0,0 +1,107 @@
+import {formatNumber, roundToPrecision} from '../dataFormatters/dataFormatters';
+import {UNBREAKABLE_GAP, isNumeric} from '../utils';
+
+import type {FormatToSizeArgs, FormatValuesArgs} from './common';
+import i18n from './i18n';
+
+const sizes = {
+    thousand: {
+        value: 1_000,
+        label: i18n('label_thousand'),
+    },
+    million: {
+        value: 1_000_000,
+        label: i18n('label_million'),
+    },
+    billion: {
+        value: 1_000_000_000,
+        label: i18n('label_billion'),
+    },
+    trillion: {
+        value: 1_000_000_000_000,
+        label: i18n('label_trillion'),
+    },
+};
+
+export type Digits = keyof typeof sizes;
+
+/**
+ * This function is needed to keep more than 3 digits of the same size.
+ *
+ * @param significantDigits - number of digits above 3
+ * @returns size to format value to get required number of digits
+ *
+ * By default value converted to the next size when it's above 1000,
+ * so we have 900mb and 1gb. To extend it additional significantDigits could be set
```

Comment:
I suppose mb and gb are not accurate examples as gb is 1024mb

Reply:
oh this terrible copy-paste -_-